Dockerfile

```diff
@@ -19,6 +19,9 @@ ENV CONSUL_PREFIX=healthcheck
 ENV CONSUL_HTTP_ADDR=""
 ENV CONSUL_HTTP_AUTH=""
 ENV EXTRA_ARGS=""
+#ENV HONEYCOMB_API_KEY="ce8f58cf494fa93b7376d053c577a894"
```

Comment:
is this an actual API key?  perhaps don’t check into GH 😄

Reply:
um... no, that couldn't be valid key... nope, nothing to see here. Thanks for point that out, I revoked that key.